networks.yaml

```diff
@@ -73,10 +73,10 @@ arbitrum-one:
       address: "289ba1701C2F088cf0faf8B3705246331cB8A839"
       startBlock: 5856156
     minterV1:
-      address: "4969dcCF5186e1c49411638fc8A2a020Fdab752E"
+      address: "c20DE37170B45774e6CD3d2304017fc962f27252"
       startBlock: 5856338
     minter:
-      address: "4969dcCF5186e1c49411638fc8A2a020Fdab752E"
+      address: "c20DE37170B45774e6CD3d2304017fc962f27252"
```

Comment:
Do we need to include both addresses so the subgraph can index events from the first version of the Minter from when it was deployed and then also index the second version of the Minter starting from when it was deployed (and no longer indexing the first version from then on?).